djangochannelsrestframework/observer/model_observer.py

```diff
@@ -184,6 +187,8 @@ def serialize(self, instance, action, **kwargs) -> Dict[str, Any]:
         message_body = {}
         if self._serializer:
             message_body = self._serializer(self, instance, action, **kwargs)
+        elif self._serializer_class:
+            message_body = self._serializer_class(instance).data
```

Comment:
could you add a test that checks this works as expected on a delete action.

Reply:
ok!